packages/nimble-components/src/anchor-tabs/specs/README.md

```diff
@@ -0,0 +1,206 @@
+
+In addition to indicating which tab is active, navigation tabs must also separately indicate which tab has keyboard focus. I.e. you can arrow through tabs without activating those tabs, and there must be a visual indication of which tab has keyboard focus. We will indicate the focused tab with a double underline similar to what the existing tab control shows when keyboard focused, but it will be two thin lines when focusing a non-selected tab. When focusing the selected tab, the two thin underlines will overlap the thicker underline indicating selection, so we end up with a visual that is consistent with our existing tab control.
+
+---
+
+## Implementation
+
+As long as there is at least one non-disabled tab in the tabs group, we will have an active tab. Upon connecting `nimble-navigation-tabs` to the DOM, we will activate (i.e. navigate to) the tab specified as initially active. If no tab was specified as initially active, we will default to the first non-disabled tab.
+
+Each `nimble-navigation-tab` must have an id so that `nimble-navigation-tabs`'s `activeid` has a valid value. If the user does not specify an id for a tab, we will generate one using FAST's `uniqueId()` utility function.
+
+If the active tab becomes disabled, it will remain active. We will not automatically activate a different tab.
+
+When the tabs group gains or loses focus we must set or remove the `focused` class from one of the child tabs. To do this, we will add `onfocus` and `onblur` event handlers to the tabs group.
+
+
+### States
+
+#### Keyboard focused state
+When tabbing to the `nimble-navigation-tabs`, it gets keyboard focus. To indicate this, the active tab (which is already underlined) gets a second, thinner underline.
+
+### Accessibility
+
+- *Keyboard Navigation and Focus*
+    -   The tabs control will have a visual indication of which tab is active (green underline).
+    -   Once focused, the tabs control will show a visual indication of which tab has focus, starting with the active tab.
+    -   Arrowing left or right will change the focused tab. Arrowing left on the first tab will wrap around to the last, and vice versa.
+    -   `Home` will move focus to the first (non-disabled) tab.
+    -   `End` will move focus to the last (non-disabled) tab.
+    -   The `Enter` or `Space` key will activate the focused tab, moving the active indicator and loading the tab's associated URL.
+- *Form Input*
+    - N/A
+- *Use with Assistive Technology*
+    - We cannot use the `tablist`, `tab`, and `tabpanel` roles because we have no `tabpanel`s. Instead we use a `nav` container element (which has role `navigation`) and `link` roles on each of our tabs. An element with role `navigation` should have a label that describes the navigation group. The user will have to provide this, so we will propagate the `aria-label` attribute value to the `nav` element in the shadow DOM.
```

Comment:
Each of the different role types have different keyboard interactions. I don't think the keyboard interactions that are defined are aligned with navigation role which I believe is just a tab key ordered list of links.

> We cannot use the `tablist`, `tab`, and `tabpanel` roles because we have no `tabpanel`s

How confident are we in that assertion? Looking at the arai tab role I'm seeing that everything related to tabpanel realtionship is a "SHOULD" not a "MUST": https://w3c.github.io/aria/#tab

Reply:
I've changed the spec so that we're using the `tablist` and `tab` roles instead of `navigation` and `link`. The latter does not have the keyboard interaction scheme we want.